renovate.json

```
@@ -2,7 +2,8 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:recommended",
-    "monorepo:lerna"
+    "monorepo:lerna",
+    ":semanticCommitsDisabled"
```

```suggestion
    ":semanticCommits"
```

If you do want Renovate do use Semantic Commits:

- [ ] Apply this suggestion
- [ ] Change the PR title so it uses Semantic Commits in your style, probably something like `chore: force Renovate to use Semantic Commits`
- [ ] Edit the PR body so it's about enabling Semantic Commits